ceph/ceph_admin/helper.py

```diff
@@ -440,6 +440,34 @@ def create_spec_file(self):
         return temp_file.name
 
 
+def create_config_file(node, config):
```

Comment:
would it make sense to rename it as `create_ceph_config_file`

Reply:
It might. I have changed it.